File: CursoJava/src/Exer/Exer06.java
import java.util.Scanner;

public class Exer06 {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		
		double a = sc.nextDouble();
		double b = sc.nextDouble();
		double c = sc.nextDouble();
		
		double triangulo = a * c / 2.0;		double circulo = 3.14159 * c * c;		double trapezio = (a + b) / 2.0 * c;		double quadrado = b * b;		double retangulo = a * b;
		
		System.out.printf("TRIANGULO: %.3f%n", triangulo);
		System.out.printf("CIRCULO: %.3f%n", circulo);
		System.out.printf("TRAPEZIO: %.3f%n", trapezio);
		System.out.printf("QUADRADO: %.3f%n", quadrado);
		System.out.printf("RETANGULO: %.3f%n", retangulo);
		
		sc.close();
	}
}

File: CursoCSharp/Exer/EstruturaSequencial/Exer06.cs
using System;

namespace exer.udemy.EstruturaSequencial
{
    public class Exer06
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Informe o primeiro valor: ");
            double a = double.Parse(Console.ReadLine());

            Console.WriteLine("Informe o segundo valor: ");
            double b = double.Parse(Console.ReadLine());

            Console.WriteLine("Informe o terceiro valor: ");
            double c = double.Parse(Console.ReadLine());

            double triangle = a * c / 2.0;
            double circle = 3.14159 * (c * c);
            double trapezoid = (a + b) / 2.0 * c;
            double square = b * b;
            double rectangle = a * b;

            Console.WriteLine("Triangulo: " + triangle.ToString("F2"));
            Console.WriteLine("Circulo: " + circle.ToString("F2"));
            Console.WriteLine("Trapezio: " + trapezoid.ToString("F2"));
            Console.WriteLine("Quadrado: " + square.ToString("F2"));
            Console.WriteLine("Retangulo: " + rectangle.ToString("F2"));
        }
    }
}